Docs/ProjectSpec.md

```diff
@@ -373,7 +377,13 @@ targets:
 
 ### Build Script
 
-Run script build phases added via **prebuildScripts** or **postBuildScripts**. They run before or after any other build phases respectively and in the order defined. Each script can contain:
+Run script build phases can be added at 3 different points in the build:
```

Comment:
I guess another option would be to have a single `buildScripts` array with each item having an `order` enum. Is this cleaner and more extensible? Thoughts?

Reply:
Since we already have the two values, I think this is cleaner. These should be the only 3 spots needed. If we did any refactor I would want to be able to specify the order of all of the phases. It's hard for me to imagine how that would look, and it's probably a big enough breaking change that it would need to be part of a 2.0 release.